index.html

```diff
@@ -0,0 +1,164 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+</head>
+<body>
+    <header class = "top">
+        <div class="nav-top">
+        <a href="#index"><img src="Logo.svg" class="logo"></a>
+        <nav class="navbar">
+            <div class="fas fa-bars menu" id="menu-icon"></div>
+        </nav>
+    </div>
+    </header>
+<section class="paragraph">
+    <h1 class="intro">I'm Shoaib
+        Glad to see you!</h1>
+        <h3 class="detail">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</h3>
+    <div class="holder">
+        <img src="Connect.svg" class="connect">
+        <ul class="links">
+            <li><a href="#"><img src="Twitter.svg" class="socialmedia"/></a></li>
+            <li><a href="#"> <img src="Linkedin.svg"class="socialmedia"/></a></li>
+            <li><a href="#"><img src="Vector-1.svg"class="socialmedia"/></a></li>
+            <li><a href="#"> <img src="github.svg"class="socialmedia"/></a></li>
+            <li><a href="#"> <img src="Vector.svg"class="socialmedia"/></a></li>
+          </ul>
+        </div>
+    </section>
```

Comment:
## Note
You submitted the review for the `Portfolio: setup and mobile first`. Since you have made all the required changes by the previous reviewer, your project is going to be approved. :smile:   Kindly note that this approval is for only the `Portfolio: setup and mobile first` and not the `Portfolio: finish mobile version` activity. 
Kindly submit a different pull request for that. :pray:

Reply:
done